modules/installation-methods-for-different-architectures.adoc

```diff
@@ -19,12 +19,12 @@
 a|Operator (preferred), Helm charts, or `roxctl` CLI (not recommended)
 
 | ppc64le ({ibm-power})
-|No
+|Yes
 |Yes ({ocp} version 4.12 and later)
-.2+a|Operator is the only supported install method.
+.2+a|Operator (preferred) or `roxctl` CLI
```

Comment:
Need someone from the IBM team to confirm that we don't support `Helm charts` as the installation method.

Reply:
We don't officially support `Helm charts` as the installation method for P&Z